src/js/actions/export.js

```diff
@@ -975,7 +975,7 @@ define(function (require, exports) {
     exports.exportLayerAssets = exportLayerAssets;
     exports.exportDocumentAssets = exportDocumentAssets;
     exports.setUseArtboardPrefix = setUseArtboardPrefix;
-    exports.afterStartup = afterStartup;
+    // exports.afterStartup = afterStartup;
```

Comment:
i'll fix this before this branch is merged, but since i dont build generator with PS it is easier for me just to comment this out while working